packages/visx-responsive/src/enhancers/withScreenSize.tsx

```diff
@@ -1,7 +1,8 @@
 import debounce from 'lodash/debounce';
 import React from 'react';
+import { Simplify } from '../types';
 
-export type WithScreenSizeProps = {
```

Comment:
wondering if we can still export this, if people are doing deep imports (`import { WithScreenSizeProps } from '@visx/responsive/lib/enhancers/withScreenSize'`) this is theoretically breaking (debatable whether it should be supported or not...)

Reply:
Yeah, good point. In general, I'm not too fond of deep imports like this and I'd discourage people from using them but you're right, we'd probably break a lot of things with this change considering using this type was the only workaround to the issue I found. I brought it back, annotated it with `@deprecated` JSDoc, and left a to-do comment to remove in the next major version.